addons/mail/tests/test_link_preview.py

```diff
@@ -86,3 +97,14 @@ def test_02_link_preview_create(self):
                     }
                 }]
             )
+
+    def test_03_link_preview_create_no_content_type(self):
+        with patch.object(requests.Session, 'head', _patched_get_no_content_type), patch.object(requests.Session, 'head', _patched_head_no_content_type):
```

Comment:
You are patching two times head ? Shouldn't you patch get in the first one ?

Reply:
Sorry @tde-banana-odoo that is my mistake, thanks for catching that.